config/v1/config_test.go

```diff
@@ -83,7 +94,7 @@ func TestParse(t *testing.T) {
 	assert.NotNil(t, c.Modules)
 }
 
-func TestJsonFailure(t *testing.T) {
+func TestYamlFailure(t *testing.T) {
```

Comment:
rather than replace json, let's do both for now

Reply:
If I want to test both I need to make a separate ReadConfig method inside config.go. Is that fine or is there a better way of going around that? @ryanking